src/common/gui/CScalableBitmap.cpp

```diff
@@ -16,13 +34,8 @@ void CScalableBitmap::setPhysicalZoomFactor(int zoomFactor)
     currentPhysicalZoomFactor = zoomFactor;
 }
 
-#if __linux__
 CScalableBitmap::CScalableBitmap(CResourceDescription desc)
-    : CBitmap(originalPlatformBitmapForID(desc.u.id))
-#else
-CScalableBitmap::CScalableBitmap(CResourceDescription desc) 
     : CBitmap(desc)
-#endif
```

Comment:
I'm surprised this diff works; On linux you will end up with a base class for this instance with size 0 won't you? So then clients of CScalableBitmap who ask ->getWidth() will get a 0 even though it has drawable size.

I know on Mac having a non-valid superclass constructor caused mis-draws and crashes, which is why I kept this here there.

Reply:
Huh cool but from your screenshot it does work. OK this one is fine!